lib/shore/customers/customer_endpoints.rb

```diff
@@ -29,19 +29,18 @@ def get_customer(oid, customer_id)
       #   after the entry with this given event id. Option ignored if
       #   blank or not a known event_id for any of the entries.
       def get_customer_feed(oid, customer_id, options = {})
-        format_response(conn.get(customer_feed_path(oid, customer_id), options))
+        conn.get(customer_feed_path(oid, customer_id), options)
       end
 
       def update_customer(oid, customer_id, attributes)
-        format_response(conn.put do |req|
-                          req.url customer_path(oid, customer_id)
-                          req.headers['Content-Type'] = 'application/json'
```

Comment:
You don't need this anymore if you added it to the headers of the `@conn` in `initialize_conn`
```
@conn.headers = { 'Content-Type' => 'application/json' }
```